WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -136,6 +136,12 @@ public struct WordPressAuthenticatorConfiguration {
     ///
     let skipXMLRPCCheckForSiteDiscovery: Bool
 
+    /// If enabled, site address login will not check for XMLRPC URL or use XMLRPC API to handle login.
+    /// Important: make sure to implement the delegate method `handleSiteCredentialLogin` when enabling this config.
+    ///
+    ///
+    let skipXMLRPCCheckForSiteAddressLogin: Bool
```

Comment:
Nit:   

This changes two things.

1. After entering the site address, we skip the XMLRPC check
1. After entering the site credentials, we skip the whole login process and delegate the work to the client app.

WDYT think about naming it as `delegateSiteAddressLogin` or `shouldDelegateSiteAddressLogin`? 

IMHO, if we use `skipXMLRPCCheck...` pattern, it might convey that we are only skipping XMLRPC check just like `skipXMLRPCCheckForSiteDiscovery`.

Reply:
Good point! I added an extra config for manual site address login in ff454bb, let me know what you think.

Reply:
Looks great. 👍